base/src/main/java/com/smartdevicelink/transport/TransportConstants.java

```diff
@@ -97,6 +97,8 @@
     public static final String SDL_NOTIFICATION_CHANNEL_NAME = "SmartDeviceLink";
     public static final String SDL_ERROR_NOTIFICATION_CHANNEL_ID = "sdl_error_notification_channel";
     public static final int SDL_ERROR_NOTIFICATION_CHANNEL_ID_INT = 99;
+    public static final String VEHICLE_INFO = "vehicle_info";
+    public static final String UNSUPPORTED_VEHICLE_INFO_REASON = "Vehicle Info is not supported";
```

Comment:
Referring to my comment on removing onError calls in `BaseLifecycleManager` please remove this as it is not needed.
```suggestion
```

Reply:
Removed (23ad06b)